### PR TITLE
Get iot devices

### PR DESCRIPTION
**Pull request by blackout1471**

### CODE REVIEW

File: docs/SSD diagrams/WebclientGetDeviceStatus.plantuml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@startuml WebclientGetDeviceStatus
+
+entity IoTDevices as client
```
Comment (Ripiter) [inline docs/SSD diagrams/WebclientGetDeviceStatus.plantuml:3]: move IoTdevice at the bottom of participants so it moves most left

File: docs/SSD diagrams/WebclientGetDeviceStatus.plantuml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+client -> mqttService: Sends status data to subscribed clients
+mqttService -> service: Call subscribed method
+service -> service: Get device type
+service -> service: Device exists in array?
+alt exists
+    service -> service: Update known device
+else
+    service -> service: push new device into array
+end
+
```
Comment (Ripiter) [inline docs/SSD diagrams/WebclientGetDeviceStatus.plantuml:21]: missing show status of devies

Comment (blackout1471) [reply docs/SSD diagrams/WebclientGetDeviceStatus.plantuml:21]: Automatic update

File: docs/SSD diagrams/WebclientGetDeviceStatus.plantuml (hunk 1 above)
Comment (blackout1471) [reply docs/SSD diagrams/WebclientGetDeviceStatus.plantuml]: Updated.

**Review by Ripiter — APPROVED** [review]
looks good

**Pull request merged**